clientconn_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	"time"
 
 	"golang.org/x/net/http2"
+	grpcbackoff "google.golang.org/grpc/backoff"
```

Comment:
Not to nit-pick, but I would prefer to rename `internal/backoff` and leave "main" packages like this not renamed where possible.

Reply:
Done.